website/static/js/home-page/newAndNoteworthyPlugin.js

```diff
@@ -0,0 +1,166 @@
+/**
+ * New and Noteworthy Projects
+ */
+
+var $ = require('jquery');
+var m = require('mithril');
+var $osf = require('js/osfHelpers');
+var Raven = require('raven-js');
+
+// CSS
+require('css/quick-project-search-plugin.css');
```

Comment:
Yes we should separate these files even if resulting files are very small. They will need one call per page in either case. @pattisdr can you take care of this?